packages/test-runner/src/runner.ts

```diff
@@ -62,17 +73,28 @@ export class Runner {
       hasSetup = hasSetup || fs.statSync(path.join(config.testDir, 'setup.ts')).isFile();
     } catch (e) {
     }
-    if (hasSetup)
+    if (hasSetup) {
+      global['setParameterValue'] = this.setParameterValue.bind(this);
+      global['setParameterValues'] = this.setParameterValues.bind(this);
+      global['before'] = (fn: Function) => this._beforeFunctions.push(fn);
+      global['after'] = (fn: Function) => this._afterFunctions.push(fn);
```

Comment:
nit: might as well remove from global after requiring setup.

Reply:
sure, but we don't run any more code after it. we need a better solution to this anyways!